cmake/utils.cmake

```diff
@@ -0,0 +1,26 @@
+include_guard()
```

Comment:
You should add a license head as other files to pass the checker.

Reply:
I'll do you a favor :)

Reply:
thx!